src/XMakeBuildEngine/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -317,7 +317,7 @@ private string GetProcessesToIgnoreKey(long hostHandshake, long clientHandshake,
         private void ValidateRemotePipeSecurityOnWindows(NamedPipeClientStream nodeStream)
         {
             SecurityIdentifier identifier = WindowsIdentity.GetCurrent().Owner;
-#if FEATURE_PIPE_SECURITY
+#if !FEATURE_PIPE_SECURITY
```

Comment:
I agree, very confusing! Maybe we should rename that to `FEATURE_USE_LOCAL_COPY_PIPE_SECURITY` or something better?